lib/zero_silent_failures/monitor.rb

```diff
@@ -16,11 +18,6 @@ def initialize(function = nil, file = nil, line = nil)
       end
     end
 
-    # create ZSF monitor instance
-    def initialize(service)
-      @service = service
-    end
-
```

Comment:
irb(main):007> ZeroSilentFailures::Monitor.new('test-application')

=> #<ZeroSilentFailures::Monitor:0x000000012b89e5f0 @service="test-application">

Confirmed this worked as intended without the explicit `initialize` method.